<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordWidget</class>
 <widget class="QWidget" name="RecordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>555</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Record</string>
  </property>
  <layout class="QHBoxLayout" name="recordMainHLayout" stretch="0,4">
   <item>
    <widget class="QGroupBox" name="recordSettingsGroup">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="recordSettingsVLayout">
      <item>
       <layout class="QVBoxLayout" name="recordInterfaceVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="recordInterfaceLbl">
          <property name="text">
           <string>Network interface</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="recordInterfaceSelect"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="recordFilteringHLayout" stretch="2,1,0,0">
        <item>
         <layout class="QVBoxLayout" name="AddressVLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="recordHostLbl">
            <property name="text">
             <string>Multicast address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="recordHost"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="recordPortVLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="recordPortLbl">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="recordPort"/>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QCheckBox" name="recordUnicastCheckBox">
          <property name="text">
           <string>Unicast</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QCheckBox" name="recordRtpFecCheckBox">
          <property name="text">
           <string>RTP-FEC</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="recordExpandPCAPFilterButton">
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string>Advanced</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="recordPcapFilterContainer" native="true">
        <layout class="QVBoxLayout" name="recordPcapFilterVLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="recordFilterLbl">
           <property name="text">
            <string>PCAP filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="recordFilter"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QChartView" name="graphView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>700</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="recordFileFormatVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="recordFileFormatHLayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="recordFilenameVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="recordFilenameLbl">
          <property name="text">
           <string>Save to file (.ts or .pcap)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="recordFilenameHLayout">
          <item>
           <widget class="QLineEdit" name="recordFilename">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="recordOpenFileDialog">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="recordStartStopBtn">
        <property name="text">
         <string>Start recording</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="recordStatusVLayout">
     <item>
      <widget class="QGroupBox" name="recordStreamInfoGroup">
       <property name="title">
        <string>Stream info</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="styleSheet">
           <string notr="true">QTreeView::branch:has-siblings:!adjoins-item {
    border-image: url(:/img/res/stylesheet-vline.png) 0;
}

QTreeView::branch:has-siblings:adjoins-item {
    border-image: url(:/img/res/stylesheet-branch-more.png) 0;
}

QTreeView::branch:!has-children:!has-siblings:adjoins-item {
    border-image: url(:/img/res/stylesheet-branch-end.png) 0;
}

QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
        border-image: none;
        image: url(:/img/res/stylesheet-branch-closed.png);
}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  {
        border-image: none;
        image: url(:/img/res/stylesheet-branch-open.png);
}</string>
          </property>
          <column>
           <property name="text">
            <string>Stream info</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="recordStatusGroup">
       <property name="title">
        <string>Status</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="recordStatus">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Idle</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
